docs/environment-workflow.mdx

```diff
@@ -248,52 +243,31 @@ Each environment is completely isolated - no conflicts, no interference.
 
 ## Best Practices
 
-<AccordionGroup>
-  <Accordion title="Start with Quick Assessment">
-    Always use `container-use diff` and `container-use log` first. Most of the time, this gives you enough information to decide next steps without the overhead of checking out or entering containers.
-  </Accordion>
-
-{" "}
-
-<Accordion title="Choose the Right Acceptance Method">
-  Use **merge** when you want to preserve the agent's commit history and
-  understand how the work evolved. Use **apply** when you want to create clean,
-  customized commits or review changes before committing.
-</Accordion>
-
-{" "}
+- **Start with Quick Assessment**: Always use `container-use diff` and `container-use log` first. Most of the time, this gives you enough information to decide next steps without the overhead of checking out or entering containers.
 
-<Accordion title="Don't Be Afraid to Delete">
-  Environments are disposable by design. If an agent gets stuck or goes down the
-  wrong path, it's often faster to delete and start fresh than to try to fix
-  problematic work.
-</Accordion>
+- **Merge vs Apply**: Use `merge` when you want to preserve the agent's commit history and understand how the work evolved. Use `apply` when you want to create clean, customized commits or review changes before committing.
 
-{" "}
+- **Don't Be Afraid to Delete**: Environments are disposable by design. If an agent gets stuck or goes down the wrong path, it's often faster to delete and start fresh than to try to fix problematic work.
```

Comment:
nit: you don't actually need to delete in order to abandon. i feel like we're a little too liberal in telling folks that they should even use `cu delete`